samples/ReverseProxy.LetsEncrypt.Sample/appsettings.json

```
@@ -0,0 +1,50 @@
+{
+  "Kestrel": {
+    "Endpoints": {
+      "Http": {
+        "Url": "http://localhost:80"
+      },
+      "Https": {
+        "Url": "https://localhost:443"
```

Localhost won't work if this site is running publicly on the internet.
```suggestion
        "Url": "http://*:80"
      },
      "Https": {
        "Url": "https://*:443"
```